app/views/users/_form.html.erb

```diff
@@ -1,6 +1,5 @@
 <%provide(:title, "Signup")%>
```

Comment:
Remove this line. It is displaying:
![Screen Shot 2021-04-29 at 9 40 34 PM](https://user-images.githubusercontent.com/74635797/116560006-8da8ac80-a933-11eb-8372-e7e2a0c4ce8b.png)